src/js/sjcl-custom/sjcl-secp256k1.js

```diff
@@ -53,7 +53,13 @@ sjcl.ecc.pointJac.prototype.add = function(T) {
 // This is a custom doubling algorithm that
 // only works for a=-3 Jacobian curve. It's much
 // faster than the generic implementation
+var neg3Doubl = sjcl.ecc.pointJac.prototype.doubl;
 sjcl.ecc.pointJac.prototype.doubl = function () {
+
+  if (!this.curve.a.equals(0)) {
+    return neg3Doubl.call(this);
```

Comment:
Good point, it's not -3 only math that's stashed away, so `neg3Doubl` is a misnomer. 

You could update the errant comments while you're here too.